.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Task
```

Comment:
The goal with this template is to get whoever is creating the template to add as much context as possible. I've seen a number of issues in the repo (including those created by me) that don't have much context or info about how to proceed. I want us to be in a place where a casual contributor can look at an issue and have at least some idea what's even required to resolve an issue.

> Tasks can have subtasks, or be part of another task. We cannot express this uniformly in GitHub, but that's not our fault.

That's why I keep pushing for tracking issues. There should be a way to view one task in the context of the larger deliverable item. I think tracking issues are the least-bad option we have on GitHub without using third-party tools.

Reply:
> The goal with this template is to get whoever is creating the template to add as much context as possible.

I undestand. The point of my comment was raising the question where the template should go. Because there are templates for users to report problems (what we classically use GitHub issues for) and then templates to elaborate a task. I argue the task template maybe should not be an issue template, but rather a section in the handbook or something. Outside contributors cannot meaningfully answer the question "what *needs* to be done*.

We can of course leave it as an issue template and then make obvious that this is for internal use, which would also be fine for me.